Extension/src/nativeStrings.json

```diff
@@ -189,5 +189,6 @@
         "hint": "This label is for the return value description for a function. Usage example: 'Returns: Area of a shape.'"
     },
     "deprecated_label": "Deprecated:",
-    "exceptions_label": "Exceptions:"
+    "exceptions_label": "Exceptions:",
```

Comment:
Are these still used?

Reply:
Oh, it is.

Reply:
Yes, these labels are used when the comment is simplified.